MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -28,6 +30,7 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		viewModel?.onLoadingStateChange = { [weak self] isLoading in
 			if isLoading {
 				self?.refreshControl?.beginRefreshing()
+                self?.errorController?.hideError()
```

This logic to hide the error when loading the feed should be in the ViewModel. See my comments regarding the `onErrorStateChange` events. So you could remove this line and make the `hideError` method private in the error controller.